### ISSUE TITLE
can't compile on prod on Angular 7

### ISSUE THREAD

**Issue by medbenmakhlouf** [DanielYKPan/date-time-picker#528]
```
ERROR in ../ng-pick-datetime-moment/moment-adapter/moment-date-time.module.ts(22,2): Error during template compile of 'OwlMomentDateTimeModule'
  Function calls are not supported in decorators but 'ɵmakeDecorator' was called in 'NgModule'
    'NgModule' calls 'ɵmakeDecorator'.

```

**Comment by antomor** [comment]
Same issue. Furthermore the source code of the [package](https://www.npmjs.com/package/ng-pick-datetime-moment) is not available. Could you give us some news please?
Thank you in advance :-)

**Comment by antomor** [comment]
After a while, I have found that `OwlMomentDateTimeModule` is included in `date-time-picker` package, but if I try to use directly, it seems that it is not exported. 
I have tried to fix it locally and after exporting the module properly, it works fine.
Anyway for now I am using `OwlNativeDateTimeModule`, but I can give you more details about the issue if necessary.

**Comment by medbenmakhlouf** [comment]
@antomor , Yes Please. Could you please add the code snippet to solve the issue ? 
Many Thanks

**Comment by antomor** [comment]
Hi @medbenmakhlouf sorry for my late answer.
I simply removed that dependency, that seems to be not supported on Angular 7.
I noticed that `moment-adapter` is included in the main package, at the path [/src/date-time/adapter/moment-adapter/
](https://github.com/DanielYKPan/date-time-picker/tree/master/src/date-time/adapter/moment-adapter), but if I try to use it directly, I get an error and the module is not found. So I modified the local version of the package to export that module and their classes to be able to use it locally.
I didn't made a PR because I end up using the `native-datetime-module` instead of `moment-datetime-module`, so I don't have the code snippet, but if you still have troubles, don't hesitate.
Happy coding :-)

**Comment by mluis** [comment]
I'm getting the same issue.

**Comment by dbusacca** [comment]
I'm getting this issue after the upgrade from Angular 7.2 to Angular 8-rc4

**Comment by thesmallbang** [comment]
I was getting this error after updating to Angular 8. The solution for me was to add a defined path to angular in the tsconfig

To be honest i'm not sure why it works but it looks like people have had this issue from time to time going back over a year now.

```
"compilerOptions": {
    "paths": {
      "@angular/*": ["../node_modules/@angular/*"]
    }
}
```

**Comment by dbusacca** [comment]
For now I've rebuilded ng-pick-datetime exporting OwlMomentDateTimeModule and removed the ng-pick-datetime-moment modules, and it works like a charm with either Angular 7 and Angular 8

**Comment by piernik** [comment]
I have the same problem as well with `angular@8`

**Comment by Dedme** [comment]
i fixed this by ensuring "ng-pick-datetime-moment" is version "1.0.7". 1.0.8 is borked

**Comment by piernik** [comment]
@Dedme It's working - thanks!

@DanielYKPan I think Your component is the best in its category. I hope You will maintain it. If not find someone who will because it will be shame to loose it.

**Comment by ips-illuminz** [comment]
I fixed this by adding into main `tsconfig.json` file:
```
"paths": {
     "@angular/*": ["node_modules/@angular/*"]
}
```
It was giving the same error when adding into `tsconfig.app.json`

**Comment by vijhhh2** [comment]
> For now I've rebuilded ng-pick-datetime exporting OwlMomentDateTimeModule and removed the ng-pick-datetime-moment modules, and it works like a charm with either Angular 7 and Angular 8

Can you provide some code snippets how did that?

**Comment by ssharma94** [comment]
> i fixed this by ensuring "ng-pick-datetime-moment" is version "1.0.7". 1.0.8 is borked

Awesome! It fixed my issue!!!
